ddp-workspace/projects/ddp-sdk/src/lib/models/analyticsEventCategories.ts

```diff
@@ -0,0 +1,9 @@
+export enum AnalyticsEventCategories {
+    ClickedCountMeIn = 'clickedCountMeIn',
+    Social = 'social',
+    CloseSurvey = 'closeSurvey',
+    SubmitSurvey = 'submitSurvey',
+    Dashboard = 'dashboard',
+    Authentication = 'authentication',
+    MailingList = 'mailingList'
+}
```

Comment:
Do we have room for modifying these strings for categories and actions?
I think the separation of actions and categories is a good idea, but the strings underneath don't quite seem to match the definition of a category or action: e.g., closeSurvey seems more like an action than a category. ClickedCountMeIn also sounds more like an action (both are verbs). 'fromMainPage' sounds more like a source than an action.
If these are the strings we are stuck using, then maybe these are just tags (primary and secondary).

Reply:
You are probably right that this is not semantically, but I split values in enums not by name, but by how they are used in GA(these values were determined for angio gen2 website)

Reply:
It is what it is!